.github/workflows/android.yml

```
@@ -25,8 +25,13 @@ jobs:
       - name: Create local.properties
         run: |
           echo "CLARITY_PROJECT_ID=${{secrets.CLARITY_PROJECT_ID}}" >> //home//runner//work//Base//Base//local.properties
+          echo "KEYSTORE_KEY_ALIAS=${{secrets.MASTER_KEY}}" >> ./local.properties
```

I assume we should do this:
```suggestion
          echo "KEYSTORE_KEY_ALIAS=${{secrets.MASTER_KEY}}" >> //home//runner//work//Base//Base//local.properties
```

---

Or something like this:
```
echo "sdk.dir=/usr/local/lib/android/sdk" > ./local.properties
echo "CLARITY_PROJECT_ID=${{secrets.CLARITY_PROJECT_ID}}" >> ./local.properties
echo "KEYSTORE_KEY_ALIAS=${{secrets.MASTER_KEY}}" >> ./local.properties
```

---

Perfect, I'll update with your last suggestion.